packages/core/src/background/BackgroundSystem.ts

```diff
@@ -68,28 +61,24 @@ export class BackgroundSystem implements ISystem<BackgroundOptions>
 
         if (options.color)
         {
-            this.color = typeof options.color === 'string'
-                ? string2hex(options.color)
-                : options.color;
+            this.color = options.color;
         }
 
         this.alpha = options.alpha;
     }
 
     /**
      * The background color to fill if not transparent
-     * @member {number}
+     * @member {number|string|number[]|Float32Array}
      */
     get color(): number
     {
-        return this._backgroundColor;
+        return this._background.toNumber();
```

Comment:
perhaps it might be good to return the original value here?

```
renderer.background.color = 'red'

console.log(renderer.background.color) // 'red'
```